server/meta.jsx

```diff
@@ -37,9 +40,22 @@ function validatePaymentsSDKUrl({ pathname, query, hash }) {
             throw new TypeError(`Unexpected non-string key for sdk url: ${ key }`);
         }
 
-        if (!val.match(/^[a-zA-Z0-9_,-@.]+$/) && !val.match(/^\*$/)) {
+        if (!val.match(/^[a-zA-Z0-9+_,-@.]+$/) && !val.match(/^\*$/)) {
             throw new Error(`Unexpected characters in query key for sdk url: ${ key }=${ val }`);
         }
+
+        if (key === SDK_QUERY_KEYS.MERCHANT_ID) {
+            const merchantValues = val.split(",");
+            Array.from(merchantValues).forEach(aMerchantValue => {
```

Comment:
Do we need the Array.from() call? I think we can remove it since `split()` will always return an array.
```suggestion
            merchantValues.forEach(aMerchantValue => {
```

Reply:
Also can we rename `aMerchantValue` to just `merchantValue`?